hlx_statics/styles/styles.css

```diff
@@ -1162,3 +1162,10 @@ main .content-header.block-display .github-actions-wrapper{
 main .no-breadcrumbs {
   justify-content: end;
 }
+
+main div.default-content-wrapper img.badgeImage {
```

Comment:
My only concern here is attaching it to the class `default-content-wrapper`. Is it the case that the badge will only appear in this class?

Reply:
Maybe need to make to more generic so be something like `main div img.badgeImage`?